### DISCUSSION TITLE
How to change initial map center coordinates

### DISCUSSION THREAD

**Discussion by PentamapFlorian** [rnmapbox/maps#2946] [Q&A]
When I start my mapbox application it shortly (possibly till my state variable is intialized) shows a map centered at the Mapbox company head quarter in San Francisco. Is it possible to change these initial coordinates?

**Comment by mfazekas** [comment]
there is `defaultSettings` on Camera

**Reply by PentamapFlorian** [reply]
Hi Miklós, thanks for your answer. I couldn't find defaultSettings on MapView but on Camera, although it seems that it is still a TODO and not implemented yet.

**Reply by mfazekas** [reply]
Yep sorry defaultSetting is on Camarea. But it should work just fine.
